documentation/feature/spectator.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: wiki
+title: Spectator
+description: A flexible spectator system
+group: feature
+parent: wiki
+---
+
+## 1. Overview
+
+The ACE3 spectator system is designed to act as a flexible and easy to configure framework. Most scenarios can be set up as desired using only the settings provided, however public functions are available for finer control of these configurable aspects.
+
+### 1.1 Spectator System
+
+The current itteration of the ACE3 spectator system only officially supports scenarios using respawn type 3 (or "BASE"). However there's nothing to stop its use alongside anything else, just be aware that it might not function entirely as expected.
+
+By default, the ACE3 spectator system does nothing - meaning existing missions will behave exactly as before. The setting `ace_spectator_onDeath` can be used to automatically put players into spectator mode each time they die.
+
+For scenario designers who seek a more advanced setup (such as multiple lives or wave respawning) the function `ace_spectator_fnc_setSpectator` is provided to transition players to/from spectator mode as desired:
```

Comment:
we're calling them "mission makers" everywhere atm.
I like "scenario designers" better, though